### ISSUE TITLE
Capture/Restore doesn't reset form with list of files correctly after submitting

### ISSUE THREAD

**Issue by backstromjoel** [ciscoheat/sveltekit-superforms#386]
- [x] Before posting an issue, read the [FAQ](https://superforms.rocks/faq) and search the previous issues.

**Description**
When submitting a form with a list of files, the list is still the same size as it was before submitting, but without any data.

Form definition...

```
const quoteForm = superForm(data.form, {
  validators: zodClient(optionalPartListSchema),
  dataType: 'json',
  onError(error) {
	  console.error(error);
	  errorMessage = $LL.SOMETHING_WENT_WRONG_DESCRIPTION({ reason: error.result.error.message });
	  errorOpen = true;
  },
  onSubmit: async ({ cancel }) => {
	  if ($form.parts.length === 0) {
		  errorMessage = $LL.MUST_ADD_PART_ALERT();
		  errorOpen = true;
		  cancel();
	  }
  }
});
const { form, capture, restore, enhance } = quoteForm;
export const snapshot = { capture, restore };
```

This is the form after submitting and routing back.
![image](https://github.com/ciscoheat/sveltekit-superforms/assets/55511080/e02682a9-6e07-4576-a67e-a056dc65dad7)

Expected behaviour is for the list to be reset to not contain any elements.

**Comment by ciscoheat** [comment]
What does the schema look like? It can be difficult for the array itself to reset, as the files are nested in the contents of the array. I can surely remove `File` objects recursively, and arrays containing files, but nested data that should be removed based on its content will probably require some manual work, in a custom `capture` function.

**Comment by ciscoheat** [comment]
At least the removal of `File` objects from snapshots is added now with 2.11.0, but I'm not sure if that solves your problem entirely?

**Comment by backstromjoel** [comment]
The fixes in 2.11.0 fixed our issues! Thanks a lot, appriciate the ultra fast fix ❤️
